index.html

```diff
@@ -0,0 +1,160 @@
+                <li><a href=""> <i class="fa-brands fa-twitter"></i></a></li>
+                <li><a href=""><i class="fa-brands fa-facebook"></i></a></li>
+            </nav>
+        </div>
+        <div class="mainnav">
+            <h1 class="main-h1"> The chance</h1>
+            <nav>
+                <li><a href="/about.html"> About</a> </li>
+                <li> Contact</li>
+                <li> Sponsor</li>
+                <li>News</li>
+                <button class="main-button"> Buy Ticket now</button>
+            </nav>
+        </div>
+
+
+        <button type="button" class="ham">
+            <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+                <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M4 6h16M4 12h16M4 18h16" />
+            </svg>
+        </button>
+
+
+
+        <div class="overlayy"></div>
+        <nav class="sideNav">
+            <button type="button" class="menuX">
+                <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+                    <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M6 18L18 6M6 6l12 12" />
+                </svg>
+            </button>
+            <ul>
+                <li><a class="menuLink" href="/about.html">About</a></li>
+                <li><a class="menuLink" href="#projects-container">Contact</a></li>
+                <li><a class="menuLink" href="#about-container">Sponsort</a></li>
+                <li><a class="menuLink main-button" href="#footer">Buy Tickets</a></li>
+            </ul>
+        </nav>
+    </header>
+
+    <section class="showcase">
+        <video class="video" autoplay loop muted="" playsinline="" data-wf-ignore="true" ata-object-fit="cover">
+            <source
+                src="https://uploads-ssl.webflow.com/6180b7ea31c5203e568d88d0/618dc8b5b5da606d4e850b59_IMG_4301-transcode.mp4"
+                data-wf-ignore="true">
+            <source
+                src="https://uploads-ssl.webflow.com/6180b7ea31c5203e568d88d0/618dc8b5b5da606d4e850b59_IMG_4301-transcode.webm"
+                data-wf-ignore="true">
+        </video>
+        <div class="overlay"></div>
+        <div class="text">
+            <h2>"HELLO! PARTY PEOPLE "</h2>
+            <h3>The CHANCE CONCERT FOR 2022 </h3>
+            <p>A Concert For All We Dont Care Where Your Coming From We Dont Discriminate Its An Event Of A Life Time
+                dont miss it </p>
+            <span class="text-p"> 2022.06.15(Web)~17(Sat)</span>
+            <div class="demo">
+                <h6> @ the pensula staduim hrq</h6>
+            </div>
+        </div>
+
+    </section>
+    <div class="section2">
+        <h1> About Program</h1>
+        <div class="guide-bar"></div>
+    </div>
+
+    <div class="section3">
+        <h2 class="sectionh1"> main speaker</h2>
+        <div class="guided-bar"></div>
+    </div>
```

Comment:
- [ ] Kindly be noticed that the columns in the main speaker section are making your project have a horizontal scroll please fix that and show the columns in a responsive way.
![Screenshot from 2022-05-19 23-23-27](https://user-images.githubusercontent.com/33752735/169408680-3c2eed1f-49a9-40a0-9e6d-689d942f2148.png)